src/components/MovieCarousel.js

```diff
@@ -0,0 +1,21 @@
+import { Dimensions, View } from 'react-native';
+import React from 'react';
+import Carousel from 'react-native-reanimated-carousel';
+import { MovieCard } from '@/components/MovieCard';
+
+export const MovieCarousel = ({ movies }) => {
+  console.log(movies);
+  return (
+    <View style={{ flex: 1 }}>
+      <Carousel
+        width={Dimensions.get('window').width}
```

Comment:
Move this into a constant so it's not rendered each time in the JSX
```suggestion
        const WINDOW_WIDTH = Dimensions.get('window').width;
```